rust/src/ftp/response.rs

```diff
@@ -22,11 +22,12 @@ use std::slice;
 
 #[repr(C)]
 pub struct FTPResponseLine {
-    code: *mut u8,     // Response code as a string (may be null)
-    response: *mut u8, // Response string
-    length: usize,     // Length of the response string
-    truncated: bool,   // Uses TX/state value.
-    total_size: usize, // Total allocated size in bytes
+    code: *mut u8,      // Response code as a string (may be null)
```

Comment:
Could you make it more Rust and have `code` be a `Vec<u8>` so that you can access its length easily as well ?

Struct could not be anylonger `repr(C)` and you would need wrapper code to get it from C...

Does it make sense ?

Reply:
Sure, you still need a new PR linking to the new SV PR with green CI then ;-)